networking_aci/plugins/ml2/drivers/mech_aci/cobra_manager.py

```diff
@@ -204,24 +206,26 @@ def ensure_static_bindings_configured(self, network_id, host_config, encap=None,
         for physdom in host_config['physical_domain']:
             self._ensure_physdom(epg, physdom, (delete and physdom in physdoms_to_clear))
 
-    def ensure_baremetal_entities(self, resource_name, pc_policy_group_name):
+    def ensure_baremetal_entities(self, policy_group, project_domain, pc_policy_group_name):
         pc_policy_group_data = ACI_CONFIG.get_pc_policy_group_data(pc_policy_group_name)
         if not pc_policy_group_data:
             return False
 
         # aep, physdom, vlan pool
-        aep = infra.AttEntityP('uni/infra', name=resource_name)
-        physdom = phys.DomP('uni', resource_name)
-        vlan_pool = fvns.VlanInstP('uni/infra', name=resource_name, allocMode="static")
-        encap_blk = fvns.EncapBlk(vlan_pool, 'vlan-1', 'vlan-4095')
+        aep = infra.AttEntityP('uni/infra', name=project_domain)
```

Comment:
This puzzled me a bit cause i could not find the link when this is called with `host_config['baremetal_resource_name']` as `project_domain` which is then used for the physdom, aep. I know see that `annotate_baremetal_info` scopes this to the project.